scripts/documentation/generate.py

```diff
@@ -167,48 +320,69 @@ def OPERATOR(self, name):
         return str(name)
 
 
-"""
-The concept:
-
-We don't want to manually keep API docs in sync with SQL files.
-
-So instead we should generate the docs from the sql/install/*.sql files.
+def parse_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        '--group', '-g',
+        nargs=2, metavar=('header', 'glob'),
+        action='append',
+        help='Globs for each group of SQL install files')
+    return parser.parse_args()
 
-We have the following things to document:
-    * Type
-    * Cast (put in same section as the type)
-    * Operator
-    * Function
-
-Some functions do not need docs, we can signal this by not applying a comment.
-This is internal operator functions for example.
-
-We should parse the sql files, extract the above, and then document all things that
-have comments applied.
-
-We should probably hardcode a list of things that should not be documented, so we can
-fail if new function are undocumented.
-
-"""
```

Comment:
I think it is useful to have a the general idea of the script documented in a comment. Could you re-insert a similar comment in the top of the file?